interface-definitions/include/firewall/common-rule.xml.i

```diff
@@ -95,6 +95,51 @@
     </constraint>
   </properties>
 </leafNode>
+<node name="ct-status">
```

Comment:
I believe `connection-status` would be a better name. Experienced Linux network admins will associate `ct` with conntrack immediately, but many VyOS users actually come from non-Linux background and that name will be opaque to them.